Meadowlark-js/backends/meadowlark-mongodb-backend/src/model/MeadowlarkDocument.ts

```diff
@@ -52,13 +52,13 @@ export interface MeadowlarkDocument extends MeadowlarkDocumentId {
    * An array of ids extracted from the ODS/API document for all externally
    * referenced documents.
    */
-  outRefs: string[];
+  outboundRefs: string[];
```

Comment:
While in mongo code renaming things, renamed "outRefs" to a less cryptic "outboundRefs"